test/configs/config.spec.ts

```diff
@@ -159,7 +159,7 @@ describe('AutoblocksConfig', () => {
       });
 
       await config.activateRemoteConfig({
-        config: {
+        remoteConfig: {
```

Comment:
hmm i think i liked just config, seems kinda redundant now since the method name is already "activateRemoteConfig"

Reply:
ya i was going back forth on this lol